src/main/java/de/unistuttgart/iste/gits/media_service/controller/MediaController.java

```diff
@@ -34,6 +34,10 @@ public List<MediaRecordDto> mediaRecordsById(@Argument List<UUID> ids) {
         return mediaService.getMediaRecordsById(ids);
     }
 
+    @QueryMapping List<List<MediaRecordDto>> mediaRecordsByContentIds(@Argument List<UUID> contentIds) {
```

Comment:
(No todo)
Usually annotations are on a seperate line (but I don't care about such things)

Reply:
Blame copilot lol. Fixed it.